gatsby/src/components/header/header-locale-select.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import classnames from 'classnames';
+
+import GovIcon from '../gov-icon';
+
+import classes from './header-locale-select.module.scss';
+
+import { locales } from './header';
+
+type Props = {
+  activeLocale: string;
+  onLocaleChange: (locale: string) => void;
+};
```

Comment:
Components should use `interface` instead of `type`

Also we are using 'gatsby-plugin-react-i18next'
Component should be able to access available languages and change language on its own
[https://www.gatsbyjs.com/plugins/gatsby-plugin-react-i18next/#changing-the-language](https://www.gatsbyjs.com/plugins/gatsby-plugin-react-i18next/#changing-the-language)

Reply:
> Also we are using 'gatsby-plugin-react-i18next'
Component should be able to access available languages and change language on its own
https://www.gatsbyjs.com/plugins/gatsby-plugin-react-i18next/#changing-the-language

I think we can skip for now as we did not decide clearly how we do translation